.eslintrc.json

```
@@ -0,0 +1,38 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true,
+        "jest": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "plugin:react/jsx-runtime",
+        "standard",
+        "prettier"
+    ],
+    "ignorePatterns": ["website/**/*", "examples/**/*", "docs-e2e-testing/**/*"],
```

Can you update it to?
```suggestion
    "ignorePatterns": ["website/**/*", "examples/**/*", "docs-e2e-testing/**/*", "transforms/**/*", "dist/**/*", "playground/**/*"],
```

---

Done